scripts/iso3166.py

```diff
@@ -0,0 +1,42 @@
+import os
+import pandas as pd
```

Comment:
I don't believe we need to use pandas which is quite huge lib. Also, it isn't listed in the requirements here https://github.com/datasets/country-codes/blob/master/scripts/requirements.pip

Reply:
Reflected in https://github.com/datasets/country-codes/pull/83/commits/698cc4f4bfe973941f0ec85318b39dd7fcc0d822 cc @gradedSystem